index.html

```diff
@@ -0,0 +1,228 @@
+        }
+        @media (max-width: 600px) {
+            body {
+                padding: 10px;
+            }
+        }
+    </style>
+</head>
+<body>
+    <h1>Virus Scan Website</h1>
+
+    <div id="upload-section">
+        <input type="file" id="file-input">
+        <button id="upload-button" disabled>Upload and Scan</button>
+    </div>
+
+    <div id="status-section" style="display: none;">
+        <h2>Scan Status</h2>
+        <p id="status-text">Uploading file...</p>
+    </div>
+
+    <div id="results-section">
+        <h2>Scan Results</h2>
+        <div id="results-content"></div>
+    </div>
+
+    <div id="recent-scans-section">
+        <h2>Recent File Scans</h2>
+        <button id="expand-recent-scans" class="expand-button">Show Recent Scans</button>
+        <div id="recent-scans-list" style="display: none;"></div>
+    </div>
+
+    <script>
+        const uploadSection = document.getElementById('upload-section');
+        const statusSection = document.getElementById('status-section');
+        const resultsSection = document.getElementById('results-section');
+        const recentScansSection = document.getElementById('recent-scans-section');
+        const fileInput = document.getElementById('file-input');
+        const uploadButton = document.getElementById('upload-button');
+        const statusText = document.getElementById('status-text');
+        const resultsContent = document.getElementById('results-content');
+        const expandRecentScansButton = document.getElementById('expand-recent-scans');
+        const recentScansList = document.getElementById('recent-scans-list');
+
+        // enable/disable upload button
+        fileInput.addEventListener('change', (event) => {
+          const file = event.target.files;
+
+          if (file) {
+              uploadButton.removeAttribute('disabled');
+          } else {
+              uploadButton.setAttribute('disabled');
+          }
+        });
+
+
+        uploadButton.addEventListener('click', () => {
+            const file = fileInput.files[0];
+            if (file) {
+                uploadSection.style.display = 'none';
+                statusSection.style.display = 'block';
+                handoffToPython(file);
+            } else {
+                alert('Please select a file to upload.');
+            }
+        });
+
+        expandRecentScansButton.addEventListener('click', () => {
+            if (recentScansList.style.display === 'none') {
+                recentScansList.style.display = 'block';
+                expandRecentScansButton.textContent = 'Hide Recent Scans';
+                loadRecentScans();
+            } else {
+                recentScansList.style.display = 'none';
+                expandRecentScansButton.textContent = 'Show Recent Scans';
+            }
+        });
+
+        function handoffToPython(file) {
+            // FIXME: trigger python file, await results
+            results = 1;
```

Comment:
results = 1 is just a placeholder until we figure out how to write back and forth between this file and the python file.